<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShipperWindow</class>
 <widget class="QDialog" name="ShipperWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>804</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>461</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelNameBook">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Book name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBookName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addBookButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>750</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>AddBook</string>
   </property>
  </widget>
  <widget class="QTableView" name="addAuthorView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>421</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>760</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>421</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="datePublic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="datePrint">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>620</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>700</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
  </widget>
  <widget class="QTableView" name="houseView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>131</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="categoryView">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>141</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="typeView">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>131</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="imageButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>770</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>image</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>410</y>
     <width>461</width>
     <height>351</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
